test/e2e/utils/constants.go

```diff
@@ -23,12 +23,14 @@ const (
 	OperatorDeploymentName = "zero-trust-workload-identity-manager-controller-manager"
 	OperatorLabelSelector  = "name=zero-trust-workload-identity-manager"
 
-	SpireServerStatefulSetName   = "spire-server"
-	SpireServerPodLabel          = "app.kubernetes.io/name=spire-server"
-	SpireAgentDaemonSetName      = "spire-agent"
-	SpireAgentPodLabel           = "app.kubernetes.io/name=spire-agent"
-	SpiffeCSIDriverDaemonSetName = "spire-spiffe-csi-driver"
-	SpiffeCSIDriverPodLabel      = "app.kubernetes.io/name=spiffe-csi-driver"
+	SpireServerStatefulSetName               = "spire-server"
+	SpireServerPodLabel                      = "app.kubernetes.io/name=spire-server"
+	SpireAgentDaemonSetName                  = "spire-agent"
+	SpireAgentPodLabel                       = "app.kubernetes.io/name=spire-agent"
+	SpiffeCSIDriverDaemonSetName             = "spire-spiffe-csi-driver"
+	SpiffeCSIDriverPodLabel                  = "app.kubernetes.io/name=spiffe-csi-driver"
+	SpireOIDCDiscoveryProviderDeploymentName = "spire-spiffe-oidc-discovery-provider"
+	SpireOIDCDiscoveryProviderPodLabel       = "app.kubernetes.io/name=spire-oidc-discovery-provider"
```

Comment:
```suggestion
	SpireOIDCDiscoveryProviderPodLabel       = "app.kubernetes.io/name=spiffe-oidc-discovery-provider"
```

If you want to validate by your self:
```
oc get po -l app.kubernetes.io/name=spiffe-oidc-discovery-provider -n zero-trust-workload-identity-manager
NAME                                                   READY   STATUS    RESTARTS   AGE
spire-spiffe-oidc-discovery-provider-9fc47c7f4-k4frt   1/1     Running   0          119s

oc get po -l app.kubernetes.io/name=spire-oidc-discovery-provider -n zero-trust-workload-identity-manager
No resources found in zero-trust-workload-identity-manager namespace.
```